pkg/manifests/manifests_test.go

```diff
@@ -437,3 +440,49 @@ func TestGetDaemonSet(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdateResourceTopologyExporterCommandMultipleCalls(t *testing.T) {
```

Comment:
this should be in `updates_test.go`

Reply:
please make sure to add the copyright boilerplate like at the beginning of `manifests_test.go`

Reply:
Done